src/test/resources/ovs/v2/EventSchema.json

```diff
@@ -327,31 +327,7 @@
         },
```

Comment:
In the row above (PublisherRole) you would maybe also want the enum values: TWG, LSH and BUK